Masquerade4997Code/src/main/java/org/firstinspires/ftc/teamcode/Robots/TestRobot/TestbotTeleop.java

```diff
@@ -12,7 +12,8 @@
     private TestRobot robot = new TestRobot();
     @Override
     public void runLinearOpMode() {
-        robot.init(hardwareMap);
+        robot.mapHardware(hardwareMap);
```

Comment:
All of these 'mapHardware' should replaced with init() because init() calls mapHardware.

Reply:
This is the testbot so I didnt change anything with that

Reply:
Ok, make the change for the actual robot. You might as well change it for testbot though